types/index.d.ts

```diff
@@ -38,6 +38,58 @@ declare function xastscript(
   name: string,
   attributes?: Attributes,
   ...children: Children[]
-): Element
+): xast.Element
+
+/**
+ * This unique symbol is declared to specify the key on which JSX children are passed, without conflicting
+ * with the Attributes type.
+ */
+declare const children: unique symbol
```

Comment:
huh, that's new https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#unique-symbol
neat!